### ISSUE TITLE
[$30][MSFT-148] [Challenge Feed] Keywords, Sub Track default text should be changed because user can't get any idea when using a screen reader

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2839]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges
2. Open the Screen Reader
3. Click Filters
4. Click Keywords/Sub Track drop downs

**Expected Result**
Keywords: Keywords and Sub Track: Sub Track

**Actual Result**
Keywords: Select... and Sub Track: Select...

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F55

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096052 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096052 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096052 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096052 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by veshu** [comment]
@r0hit-gupta Any update on this. Thanks!

**Comment by r0hit-gupta** [comment]
@veshu working on it too but fixing previous PRs on priority.

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2839 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2839 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097394 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097394 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by codeMinter** [comment]
@r0hit-gupta if we have pending PR feedback, we cannot take new ticket, hence opening this for others. Please pick up new tickets once all PR feedback is addressed. Thanks

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097394 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by drasticdpk** [comment]
Operating System: Windows
Browser: Chrome
URL : https://test-community-app.topcoder-dev.com/challenges

**Comment by drasticdpk** [comment]
Verified on 
Operating System: Windows
Browser: Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097394<br/><br/>```This is an automated message for crazyk via Topcoder X```
